### PR TITLE
Add GD32VF103 boards

### PR DESCRIPTION
**Pull request by wychlw**
MCU200T是纯FPGA，DDR200T同款FPGA外挂GD32

### CODE REVIEW

File: Longan_Nano/FreeRTOS/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- 下载链接：
+    - SDK：https://github.com/Nuclei-Software/nuclei-sdk
+    - toolchain：https://www.nucleisys.com/download.php
+        - https://download.nucleisys.com/upload/files/toolchain/gcc/nuclei_riscv_newlibc_prebuilt_linux64_nuclei-2024.tar.bz2
+    - openocd：https://www.nucleisys.com/download.php
+        - https://download.nucleisys.com/upload/files/toolchain/openocd/nuclei-openocd-2024.02.28-linux-x64.tgz
+
+### 硬件信息
+
+- RV-STAR 开发板（GD32VF103VBT6）
```
Comment (KevinMX) [inline Longan_Nano/FreeRTOS/README.md]: RV-STAR?

File: Longan_Nano/FreeRTOS/README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+# FreeRTOS RV-STAR 测试报告
```
Comment (KevinMX) [inline Longan_Nano/FreeRTOS/README.md]: RV-STAR?

File: Longan_Nano/RT-Thread/README.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
+# RT-Thread RV-STAR 测试报告
```
Comment (KevinMX) [inline Longan_Nano/RT-Thread/README.md]: RV-STAR?

File: Longan_Nano/RT-Thread/README.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+- 下载链接：
+    - SDK：https://github.com/Nuclei-Software/nuclei-sdk
+    - toolchain：https://www.nucleisys.com/download.php
+        - https://download.nucleisys.com/upload/files/toolchain/gcc/nuclei_riscv_newlibc_prebuilt_linux64_nuclei-2024.tar.bz2
+    - openocd：https://www.nucleisys.com/download.php
+        - https://download.nucleisys.com/upload/files/toolchain/openocd/nuclei-openocd-2024.02.28-linux-x64.tgz
+
+### 硬件信息
+
+- RV-STAR 开发板（GD32VF103VBT6）
```
Comment (KevinMX) [inline Longan_Nano/RT-Thread/README.md]: Same here

File: README.md [hunk 5]
```diff
@@ -34,7 +34,10 @@
...
 | CH592X   | [CH592X-EVT][CH592X]             | N/A        | N/A           | N/A    | N/A     | N/A    | N/A        | N/A     | N/A         | N/A       | N/A       | N/A      | N/A    | N/A        | N/A        | N/A     | N/A       | N/A         | Basic    | Basic     | N/A    | N/A     | N/A     | N/A     |
-| GD32VF103| [Longan Nano][Longan_Nano]       | N/A        | N/A           | N/A    | N/A     | N/A    | N/A        | N/A     | N/A         | N/A       | N/A       | N/A      | N/A    | N/A        | N/A        | N/A     | N/A       | N/A         | N/A      | N/A       | Basic  | N/A     | N/A     | N/A     |
+| GD32VF103| [Longan Nano][Longan_Nano]       | N/A        | N/A           | N/A    | N/A     | N/A    | N/A        | N/A     | N/A         | N/A       | N/A       | N/A      | N/A    | N/A        | N/A        | N/A     | N/A       | N/A         | CFT      | CFT       | Basic  | N/A     | N/A     | N/A     |
```
Comment (KevinMX) [inline README.md:37]: Zehpyr Basic -> CFT?

**Review by KevinMX — CHANGES_REQUESTED** [review]
Some small typos found?
Sorry for the late review :)

File: README.md (hunk 5 above)
Comment (wychlw) [reply README.md:37]: Longan的Zephyr确实是之前测过的，这次和别的103一起加了前两个

File: Longan_Nano/RT-Thread/README.md (hunk 4 above)
Comment (wychlw) [reply Longan_Nano/RT-Thread/README.md]: fixed all above

**Review by KevinMX — APPROVED** [review]

**Pull request merged**